modAionImpl/src/org/aion/zero/impl/sync/TaskImportBlocks.java

```diff
@@ -590,7 +594,7 @@ private ImportResult importBlock(AionBlock b, String displayId, PeerState state)
         }
         // trigger compact when IO is slow
         // 1 sec import time and more than 10 min since last compact
-        if (t2 - t1 > SLOW_IMPORT_TIME && t2 - lastCompactTime > COMPACT_FREQUENCY) {
+        if (t2 - t1 > this.slowImportTime && t2 - lastCompactTime > this.compactFrequency) {
```

Comment:
should delete the "1 sec import time" comment since it's configurtable now

Reply:
Also, to make the feature more user friendly, please split the log message into:
```java
log.info("Compacting state database due to slow IO time.");
t1 = System.currentTimeMillis();
this.chain.compactState();
t2 = System.currentTimeMillis();
log.info("Compacting state completed in {} ms.", t2 - t1);
```